src/app.rs

```diff
@@ -34,52 +34,37 @@ fn local_storage() -> Option<Storage> {
     }
 }
 
+const THEME_SOURCE_KEY: &str = "theme";
```

Comment:
It's better to prefix local storage keys with the "application" name. Especially when you test with localhost (which I do a lot with rustdoc). Do you mind putting back the prefix please?

Reply:
Will do. :+1: